bin/webpack.js

```diff
@@ -357,7 +357,7 @@ For more information, see https://webpack.js.org/api/cli/.`);
 				});
 
 				ifArg("display-entrypoints", function(bool) {
-					if (bool) outputOptions.entrypoints = true;
+					outputOptions.entrypoints = !!bool;
```

Comment:
Double boolean trick is enough to flip it correctly. Used it myself, but it might be some pitfalls, so might be better to do a strict equal instead `if (bool === true)`

Reply:
Yes, The trick is redundant。
The param:`bool` is true or false anyway!

Reply:
How come? I think there's something fishy here, otherwise a simple `if (bool)` would be enough, unless `bool` has values different from `true` or `false`, that's why I asked what's its value